development/build_osc_docker_image.md

```diff
@@ -0,0 +1,107 @@
+# Building OSC Docker Image
+This document describes how to setup a local build environment to build OSC as a Docker image in a tar format.
+
+## Prerequisites
+OSC image is built on Ubuntu Linux Operating system. Java related application in OSC can be compiled using Maven in windows using CYGWIN tool.  Following tools and packages are required and mandatory for OSC image builds.
+
+1. Any Linux OS distribution
+2. Java 8
+3. Apache Maven 3.3
+4. Apache Ant 1.9.X
+5. Docker 17.12.x
+5. [Access to OSC source code](./repo_access.md)
+
+## Assumptions
+This document assumes that user is performing following installation steps on Ubuntu 16.04 LTS Linux OS. And examples in this section are with reference to Ubuntu Desktop Linux OS.
+
+## Installation Steps
+### 1 : Install Java
+Install Java on your system also make sure to installed JDK and JRE both. For example to install Java Development Kit on a local Ubuntu Linux OS. [Visit install Java 8 on Ubuntu](https://tecadmin.net/install-oracle-java-8-ubuntu-via-ppa/).  
+Verify Java version and make sure the version is *1.8.xxx*.  
+![](./images/java-version.png)
+### 2 : Install Apache Maven  
+After verifying java version on your system. Download Apache Maven from its official website or use following command to download Apache Maven 3.3.9.  
+`$ cd /usr/local`  
+`$ wget http://www-eu.apache.org/dist/maven/maven-3/3.3.9/binaries/apache-maven-3.3.9-bin.tar.gz`  
+Now extract downloaded archive using following command.
+
+`$ sudo tar xzf apache-maven-3.3.9-bin.tar.gz`  
+`$ sudo ln -s apache-maven-3.3.9 apache-maven`  
+### 3 : Install Apache Ant
+Make sure you have a Java environment installed before installing  Ant.  
+Following command is an example to download Ant on Ubuntu14.04:
+```sh
+$ sudo apt-get update
+$ sudo apt-get install ant
+```
+Refer to your Linux OS guide on how to download Ant.
+
+### 4 : Install Docker
+Install Docker on your system. [Visit install Docker on Ubuntu.](https://docs.docker.com/engine/installation/linux/docker-ce/ubuntu/#install-docker-ce)

```

Comment:
RH uses Federo, we need to raise US and check with RH if we need to adapt to Federo.